.github/workflows/payment-paypal-ci.yaml

```diff
@@ -0,0 +1,63 @@
+name: payment-paypal service ci
+
+on:
+  push:
+    branches: ["main"]
+    paths:
+      - "payment-paypal/**"
+      - ".github/workflows/actions/action.yaml"
+      - ".github/workflows/payment-paypal-ci.yaml"
+  pull_request:
+    branches: ["main"]
+    paths:
+      - "payment-paypal/**"
+      - ".github/workflows/actions/action.yaml"
+      - ".github/workflows/payment-paypal-ci.yaml"
+
+  workflow_dispatch:
+
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # Shallow clones should be disabled for a better relevancy of analysis
+      - uses: ./.github/workflows/actions
+      - name: Run Maven Build Command
+        run: mvn clean install -DskipTests -f payment-paypal
+      - name: Analyze with sonar cloud
+        if: ${{ github.event.pull_request.head.repo.full_name == github.repository || github.ref == 'refs/heads/main' }}
+        env:
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        run: mvn org.sonarsource.scanner.maven:sonar-maven-plugin:sonar -f payment-paypal
+      - name: Log in to the Container registry
+        if: ${{ github.ref == 'refs/heads/main' }}
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build and push Docker images
+        if: ${{ github.ref == 'refs/heads/main' }}
+        uses: docker/build-push-action@v3
+        with:
+          context: ./payment-paypal
+          push: true
+          tags: ghcr.io/nashtech-garage/yas-payment-paypal:latest
+
+  Unit-Tests:
+    needs: Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/workflows/actions
+      - name: Run Maven Test
+        run: mvn test -f payment-paypal
+      - name: Unit Test Results
+        uses: dorny/test-reporter@v1
+        if: always()
+        with:
+          name: Unit-Test-Results
```

Comment:
Rename to a specific name such as `Payment-Paypal-Unit-Test-Results` so that it can show report after this workflow ran.

Reply:
thanks